docs/changelog/116712.yaml

```
@@ -0,0 +1,10 @@
+pr: 116712
+summary: Remove apm_user role
+area: Security
+type: breaking
+issues: []
+breaking:
+  title: Remove deprecated `apm_user` role
+  area: Authorization
+  details: The deprecated `apm_user` role has been removed.
+  impact: Users depending on built-in `apm_user` role will lose access to APM UI and data. In order to retain access, replace it by one of the existing [built-in roles](elasticsearch/roles.md): `viewer`, `editor` or `superuser`).
```

nit: leftover `)` character
```suggestion
  impact: Users depending on built-in `apm_user` role will lose access to APM UI and data. In order to retain access, replace it by one of the existing [built-in roles](elasticsearch/roles.md): `viewer`, `editor` or `superuser`.
```